doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -337,35 +345,35 @@ The examples below demonstrate the use of these variables and methods.
 Set component definition and memory ranges
 ------------------------------------------
 
-In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/main/ncs/app_envelope.yaml.jinja2>`__), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
+In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/v2.7.0/ncs/app_envelope.yaml.jinja2>`__), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
 
 .. code-block::
 
     suit-components:
     - - MEM
-    - ``{{ app['dt'].label2node['cpu'].unit_addr }}``
-    - ``{{ get_absolute_address(app['dt'].chosen_nodes['zephyr,code-partition']) }}``
-    - ``{{ app['dt'].chosen_nodes['zephyr,code-partition'].regs[0].size }}``
+    - ``{{ application['dt'].label2node['cpu'].unit_addr }}``
+    - ``{{ get_absolute_address(application['dt'].chosen_nodes['zephyr,code-partition']) }}``
+    - ``{{ application['dt'].chosen_nodes['zephyr,code-partition'].regs[0].size }}``
 
 .. note::
    See the :ref:`ug_suit_dfu_component_def` page for a full list and table of the available customizable components.
 
 Set integrated payload
 ----------------------
 
-In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/main/ncs/app_envelope.yaml.jinja2>`__, the integrated payload definition is done using the target name and binary location:
+In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/v2.7.0/ncs/app_envelope.yaml.jinja2>`__, the integrated payload definition is done using the target name and binary location:
```

Comment:
@robertstypa  A reminder as we discussed - we will need a tag v2.7.0 in the suit-generator